utils_nlp/models/transformers/sequence_classification.py

```diff
@@ -1,2 +1,99 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
+
+
+from enum import Enum
+
+from pytorch_transformers.modeling_bert import (
+    BERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    BertForSequenceClassification,
+)
+from pytorch_transformers.modeling_distilbert import (
+    DISTILBERT_PRETRAINED_MODEL_ARCHIVE_MAP,
+    DistilBertForSequenceClassification,
+)
+from pytorch_transformers.modeling_roberta import (
+    ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP,
+    RobertaForSequenceClassification,
+)
+from pytorch_transformers.modeling_xlnet import (
+    XLNET_PRETRAINED_MODEL_ARCHIVE_MAP,
+    XLNetForSequenceClassification,
+)
+
+from utils_nlp.dataset.pytorch import SCDataSet
+from utils_nlp.models.transformers.common import fine_tune
+from utils_nlp.models.transformers.common import TOKENIZER_CLASS
+
+
+MODEL_CLASS = {}
+MODEL_CLASS.update({k: BertForSequenceClassification for k in BERT_PRETRAINED_MODEL_ARCHIVE_MAP})
+MODEL_CLASS.update(
+    {k: RobertaForSequenceClassification for k in ROBERTA_PRETRAINED_MODEL_ARCHIVE_MAP}
+)
+MODEL_CLASS.update({k: XLNetForSequenceClassification for k in XLNET_PRETRAINED_MODEL_ARCHIVE_MAP})
+MODEL_CLASS.update(
+    {k: DistilBertForSequenceClassification for k in DISTILBERT_PRETRAINED_MODEL_ARCHIVE_MAP}
+)
+
+
+def list_supported_models():
```

Comment:
What do you think about making this a method of SequenceClassifier?

Reply:
That would be a good idea to quickly see models that can or cannot be used.

Reply:
Only issue is that this might be called before the creation of a classifier. Might be needed for the processor as well. I think I can add it to both classes.

Reply:
If you make it a class method, you can call it by SequenceClassifier.list_supported_models(). You can have a single internal function and call it from the class methods of both classes. 
Just thinking it makes it easier to discover than a standalone function.